### DISCUSSION TITLE
Add Comments Section in Repository

### DISCUSSION THREAD

**Discussion by JohansenJunias20** [community/community#37031] [Repositories]
Add comment sections so developer can see the review of the source code

**Comment by ossanna16** [comment]
Hi there @JohansenJunias20 and welcome to our community 😄. Can you please elaborate what you mean? We appreciate the feedback!

**Reply by JohansenJunias20** [reply]
I often look for a suitable library to meet the needs of my project.
GitHub already provides many kinds of open-source libraries but sometimes I get confused about which one to choose because they all similar. if there are comments or reviews this section will definitely help me and other developers to see how the community evaluates and reacts to these repositories

**Comment by myselfahmad** [comment]
Very good effort[.](https://contadorpalabras.org/) keep it up.
